pages/ksphere/kommander/1.1/clusters/attach-cluster/federated-addons/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: layout.pug
+navigationTitle: Federated Addons
+title: Federated Addons
+menuWeight: 7
+excerpt: A description of how federated addons work
+---
+
+## Addons federation
+
+When attaching a cluster, Kommander federates certain addons on the newly attached cluster. To customize the federation of the logging and monitoring stacks, operators can use federation labels on addons.
+
+On attachment, two factors impact if an addon is deployed or not on the attached cluster:
+1. Is the attached cluster a Konvoy cluster or not (e.g. a cluster deployed using AWS EKS)?
+2. Does a label regarding the federation of the addon exist and has it been set? If yes, what is its value?
+
+The tables below describe the list of addons and cluster addons that get federated on attachment with its related federation label (if available). Addons that do not have a federation label are federated by default. If an addon is described as only federated on non-Konvoy clusters, it will not get installed even if its federation label is set to `true`.
```

Comment:
```suggestion
The tables below describe the list of addons that get federated on attachment with its related federation label (if available). Addons that do not have a federation label are federated by default. If an addon is described as only federated on non-Konvoy clusters, it will not get installed onto Konvoy clusters even if its federation label is set to `true`.
```
I assume mentioning the difference between namespaced addons and cluster-level addons would just confuse customers.

Reply:
I have applied the change at the end of the paragraph but kept the "list of addons and cluster addons" as the tables differentiate them below.